modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -5,157 +5,292 @@
+A lost server will still be in the `system` database's view of the cluster, but in an unavailable state.
+Furthermore, according to the view of the cluster, these lost servers are still hosting the databases they had before they became lost.
+Therefore, informing the cluster of servers which are lost is not enough.
+The databases hosted on lost servers also need to be moved onto available servers in the cluster, before the lost servers can be removed.
 
-. Run `SHOW SERVERS`.
-If *all* servers show health `AVAILABLE` and status `ENABLED` continue to xref:clustering/disaster-recovery.adoc#recover-databases[Recover databases].
+==== Example verification
+The cluster's view of servers can be seen by listing the servers, see xref:clustering/servers.adoc#_listing_servers[Listing servers] for more information.
+The state has been verified if *all* servers show `health` = `AVAILABLE` and `status` = `ENABLED`.
+
+[source, cypher]
+----
+SHOW SERVERS;
+----
+
+==== Path to correct state
+The following steps can be used to remove lost servers and add new ones to the cluster.
+That includes moving any potential database allocations from lost servers to available servers.
+These steps might also recreate some databases, since a database which has lost a majority of its primary allocations cannot be moved from one server to another.
+
+.Guide
+[%collapsible]
+====
 . For each `UNAVAILABLE` server, run `CALL dbms.cluster.cordonServer("unavailable-server-id")` on one of the available servers.
-. For each `CORDONED` server, run `DEALLOCATE DATABASES FROM SERVER cordoned-server-id` on one of the available servers.
-. For each server that failed to deallocate with one of the following messages:
-.. `Could not deallocate server(s) 'serverId'. Unable to reallocate 'DatabaseId.\*'. +
-Required topology for 'DatabaseId.*' is 3 primaries and 0 secondaries. +
-Consider running SHOW SERVERS to determine what action is suitable to resolve this issue.`
+This prevents new database allocations from being moved to this server.
+. For each `CORDONED` server, make sure a new *unconstrained* server has been added to the cluster to take its place, see xref:clustering/servers.adoc#cluster-add-server[Add a server to the cluster] for more information.
+If servers were added in the 'System database write availability' step of this guide, additional servers might not be needed here.
+It is important that the new servers are unconstrained, or deallocating servers might be blocked even though enough servers were added.
 +
-or
+[NOTE]
+=====
+While recommended, it is not strictly necessary to add new servers in this step.
+However, not adding new servers reduces the capacity of the cluster to handle work.
+Furthermore, it might require the topology for a database to be altered to make deallocating servers and recreating databases possible.
+=====
+
+. For each stopped database (`currentStatus`= `offline`), start them by running `START DATABASE stopped-db`.
```

Comment:
I agree that this is good context to give, however, I don't think it belongs in this section since it would then describe the overall goal in the middle of the steps to reach that goal. I have added some sentences to the beginning of the steps to try and explain this. There is a comment there as well, so I'll try to clarify the motivation there.